cmd/lookout/event.go

```diff
@@ -104,6 +107,8 @@ func (c *EventCommand) makeDataServerHandler() (*lookout.DataServerHandler, erro
 		dataService = bblfsh.NewService(dataService, dataService, bblfshConn)
 	}
 
+	dataService = purge.NewService(dataService, dataService)
```

Comment:
This code takes some time to understand why the same variable name is used 3 times in a row. 

One possible way to change it could be by introducing a convenience method with a single argument:
```
purgedDataService := purge.NewServiceUsingSame(dataService)
```

In general, for all the services as far as I can see - `NewService` method with 2 arguments is used only in tests, the rest or the code passes same instances in both.

Although I can see benefits of having those 2 separated, on the API level it could be worth changing to:
 - exported version of `NewService(s)` receives 1 argument and delegates to
 - `newServicesUsingDifferent(s, s)`
 - which can also be used in all the tests

What do you think?

Reply:
I don't think refactoring of constructors is in a scope of this PR.